javascript/javascript2/week2/doublingnumbers.js

```diff
@@ -0,0 +1,21 @@
+// #1 Doubling the odd numbers in an array:
+
+// const numbers = [1, 2, 3, 4];
```

Comment:
don't leave long comments in the code. If you want to keep the code "Just in case", you can always get it from Git history